### DISCUSSION TITLE
Installing the plumed-devel version

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/deepmd-kit#2161] [Q&A] [answered]
Hi Guys! I want to install my plumed-devel (some codes are personally changed) with dp-lammps. I have installed the latest deepmd-kit-2.1.5 with conda (I know that plumed-2.8.0 is also installed by conda). I see the discussions in https://github.com/deepmodeling/deepmd-kit/discussions/280. I am not sure how to install my local plumed-devel package to replace the plumed-2.8.0, using conda install /mypath/plumed-devel ? Thanks in advance!

**Comment by njzjz** [comment, marked as answer]
I would recommend naming your conda package `plumed` as well, so when you install your package, the old plumed package will be automatically removed.

**Reply by Zhang-pchao** [reply]
Here are my steps:
1. git clone https://github.com/deepmd-kit-recipes/plumed-feedstock.git
2. change the source path from original url to /mypath/plumed in /mypath/plumed-feedstock/recipe/meta.yaml
3. conda install conda-build, and execute conda build . -c deepmodeling in the recipe directory
4. conda install plumed --use-local
However, the new functions of my developed plumed don't work. Are there someting wrong in above steps? Thanks again!

**Reply by Zhang-pchao** [reply]
It works after changing something in meta.yaml

**Reply by njzjz** [reply]
`--force-reinstall`
See [conda install documentation](https://conda.io/projects/conda/en/latest/commands/install.html).
